src/containers/SearchPage/SearchPage.jsx

```diff
@@ -15,17 +15,33 @@ import * as actions from './searchActions';
 import { ArticleResultShape } from '../../shapes';
 import { getResults, getLastPage, getSearching } from './searchSelectors';
 import { getLocale } from '../../modules/locale/locale';
-import SearchForm from './components/SearchForm';
-import SearchResultList from './components/SearchResultList';
 import SelectSearchSortOrder from './components/SelectSearchSortOrder';
-import SelectArticleType from './components/SelectArticleType';
+import SearchList from './components/SearchList';
+import SearchTabs from './components/SearchTabs';
 import { toSearch } from '../../util/routeHelpers';
 
 class SearchPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedIndex: 0,
+    };
+  }
   componentWillMount() {
     const { location, search } = this.props;
     if (location.search) {
       search(location.search);
+      const query = queryString.parse(location.search);
+      switch (query.articleTypes) {
```

Comment:
Kan ikke dette gjøres i SearchTabs. Føles feil å ha selectedIndex i to komponenter?